.github/workflows/run-network-tests.yml

```diff
@@ -171,9 +179,13 @@ jobs:
           docker load --input joystream-node-docker-image.tar.gz
           docker images
       - name: Install packages and dependencies
-        run: yarn install --frozen-lockfile
-      - name: Ensure query-node builds
-        run: yarn workspace query-node-root build
+        run: |
+          yarn install --frozen-lockfile
+          yarn workspace @joystream/types build
+          yarn workspace @joystream/cd-schemas generate:all
+          yarn workspace @joystream/cd-schemas build
+          yarn workspace query-node-root build
+          yarn workspace @joystream/cli build
```

Comment:
Is there a reason for building the CLI here?

Reply:
I don't think query-node tests currently use the cli so will remove

Reply:
done in https://github.com/Joystream/joystream/pull/2302/commits/d9d6762b817b9d1e8c4867bbaa7379499dfcf9c6